index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>CV</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <header>
+    <h1>Vladislav Lisetskii</h1>
+    <hr>
+    <div class="info">
+      <div class="personal-info">
+        <ul>
+          <li>Position: Fullstack developer (Java/Javascript)</li>
+          <li>Age: 25</li>
+          <li>City: Moscow</li>
+        </ul>
+      </div>
+      <div class="contacts">
+        <ul>
+          <li>E-mail: <a href="mailto:vladlis54@gmail.com">vladlis54@gmail.com</a></li>
+          <li>Phone: <a href="phoneto:88005553535">8 (800) 555-35-35</a></li>
```

Comment:
>phoneto:
I believe you mean `tel:` or does your phone support phoneto as well? :thinking:

Reply:
sure, it has to be `tel:`   ))